lms/static/js/dashboard/views/course_list_tabbed_view.js

```diff
@@ -0,0 +1,99 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone', 'underscore', 'gettext',
+        'js/dashboard/routers/dashboard_router',
+        'js/dashboard/models/settings',
+        'js/dashboard/collection/courses',
+        'js/dashboard/views/course_list_view',
+        'js/dashboard/views/course_view',
+        'js/components/tabbed/views/tabbed_view'
+    ], function (Backbone, _, gettext, DashboardRouter, Settings, CourseCollection,
+                 CourseListView, CourseView, TabbedView) {
+
+        var CourseListTabbedView = Backbone.View.extend({
+
+            el: '.wrapper-header-courses',
+
+            courses: {
+                current: {
+                    title: gettext('Current'),
+                    url: '/courses/current',
+                    index: 0
+                },
+                archived: {
+                    title: gettext('Archived'),
+                    url: '/courses/archived',
+                    index: 1
+                }
+            },
+
+            initialize: function (currentCourses, archivedCourses, courseSettings) {
+
+                var router = new DashboardRouter(),
+                    dispatcher = _.clone(Backbone.Events),
+                    settingsModel = new Settings(courseSettings);
+
+                dispatcher.listenTo(router, 'goToTab', _.bind(function (tab) {
+                    this.goToTab(tab);
+                }, this));
+
+                this.setElement(this.el);
+
+                this.currentCourseListView = new CourseListView({
+                    template: this.courses.current.template,
+                    itemViewClass: CourseView,
+                    parent: this,
```

Comment:
Don't pass `parent`. Doing so violates separation of concerns. If you need to deal with delayed rendering, initiate it from the top-down rather than bottom-up. In other words, `TabbedView` should render tabs, which should render individual course items.